### ISSUE TITLE
Does keras has a default implementation for ExponentialMovingAverage durning trainning process?

### ISSUE THREAD

**Issue by pengshuang** [keras-team/keras#10839]
My backend is tensorflow. When I trained my deep learning model using f1-score as metric, I noticed a clear discrepancy in f1-score between training log and real prediction for the same validation data. I suspected that in the training log, it used the model with origin variable and when I used the trained model to predict it used the model with ExponentialMovingAverage variable.
Is it true?
